ui/views/mus/desktop_window_tree_host_mus.h

```diff
@@ -161,7 +161,14 @@ class VIEWS_MUS_EXPORT DesktopWindowTreeHostMus
 
   std::unique_ptr<ui::EventHandler> non_client_window_event_filter_;
 
+  // In Ozone/Linux, it is not Ash who fills in window decoration
+  // (like in ChromeOS). Hence, we forcily disable 'auto_update_client_area_'
+  // in order to built-in window decorations to be provided.
+#if defined(OS_LINUX) && defined(USE_OZONE) && !defined(OS_CHROMEOS)
+  bool auto_update_client_area_ = false;
```

Comment:
Maybe better to use DesktopWindowTreeHostMus::set_auto_update_client_area and
set it to false during after creation of DWTHM in DesktopNativeWidgetAura::InitNativeWidget?

What is more, I don't like the blue theme. Seems like we use CustomeViewFrame, whereas non-ozone x11 uses NativeFrameView. Don't know if that's an issue, but still. Or maybe the difference is in different themes used? Have you ever had a chance to check Widget::GetNativeTheme()?

Reply:
Right. It is possible to call set_auto_update_client_area every time we create DesktopWindowTreeHostMus, too. Ideally also guarded with #if linux && ozone && !chrome checks.

The blue theme is like Windows-looking indeed. The first foal would at least make task manager more functional in terms of windowing functionalities. In the long run, we should implement a NonClientFrameView derived class for external window mode, and make DesktopWindowTreeHostMus use it.

NativeFrameView is actually a dummy implementation, use in case of X11 for when system decoration is going to be used, in fact.